class.jetpack-connection-banner.php

```diff
@@ -25,6 +25,39 @@ private function __construct() {
 	}
 
 	/**
+	 * Checks whether the connection banner A/B test should be ran.
```

Comment:
Can we add a comment somewhere about the actual date it's expiring? so that other people don't have to know how to read `gmmktime()`